ERCS/erc-7007.md

```diff
@@ -4,7 +4,8 @@ title: Verifiable AI-Generated Content Token
 description: An ERC-721 extension for verifiable AI-generated content tokens using Zero-Knowledge and Optimistic Machine Learning techniques
 author: Cathie So (@socathie), Xiaohang Yu (@xhyumiracle), Conway (@0x1cc), Lee Ting Ting (@tina1998612), Kartin <kartin@hyperoracle.io>
 discussions-to: https://ethereum-magicians.org/t/eip-7007-zkml-aigc-nfts-an-erc-721-extension-interface-for-zkml-based-aigc-nfts/14216
-status: Review
+status: Last Call
+last-call-deadline: 2024-06-30
```

Comment:
I'd recommend setting this to somewhere in late September, or allowing Maintainers to edit your pull request. Alternatively, you could come on our biweekly Office Hours call.

Reply:
We have edited the deadline to the end of September. Unfortunately, we could not enable maintainers' editing since the forked repo is under a GitHub org (see https://github.com/orgs/community/discussions/5634)